package.json

```diff
@@ -67,66 +67,78 @@
+    "@babel/plugin-proposal-private-property-in-object": "^7.16.7",
+    "@babel/plugin-transform-runtime": "^7.16.7",
+    "@babel/preset-env": "^7.16.7",
     "@babel/preset-react": "^7.12.5",
-    "@storybook/addon-backgrounds": "6.3.12",
+    "@storybook/addon-backgrounds": "6.4.9",
     "@storybook/addon-controls": "^6.2.8",
     "@storybook/addon-docs": "^6.2.8",
-    "@storybook/addon-knobs": "6.2.9",
+    "@storybook/addon-knobs": "6.4.0",
     "@storybook/addon-postcss": "^2.0.0",
     "@storybook/addons": "6.4.9",
-    "@storybook/react": "6.3.12",
+    "@storybook/react": "6.4.9",
     "@storybook/theming": "^6.2.8",
     "@storybook/ui": "6.4.9",
     "babel-eslint": "^10.0.2",
     "babel-loader": "^8.0.0-beta.6",
     "babel-plugin-transform-react-remove-prop-types": "^0.4.13",
-    "chromatic": "^6.0.6",
-    "css-loader": "^5.0.2",
+    "chromatic": "^6.3.3",
+    "css-loader": "^6.5.1",
     "cssnano": "^5.0.1",
-    "eslint": "^7.1.0",
+    "eslint": "^8.6.0",
     "eslint-config-prettier": "^8.0.0",
     "eslint-config-standard": "^16.0.2",
     "eslint-config-standard-jsx": "^10.0.0",
     "eslint-config-standard-react": "^11.0.1",
     "eslint-plugin-babel": "^5.3.0",
-    "eslint-plugin-import": "^2.18.2",
+    "eslint-plugin-import": "^2.25.4",
     "eslint-plugin-node": "^11.0.0",
     "eslint-plugin-prettier": "^4.0.0",
-    "eslint-plugin-promise": "^5.1.0",
-    "eslint-plugin-react": "^7.14.3",
+    "eslint-plugin-promise": "^6.0.0",
+    "eslint-plugin-react": "^7.28.0",
     "eslint-plugin-standard": "^5.0.0",
     "extract-text-webpack-plugin": "^4.0.0-beta.0",
     "file-loader": "^6.0.0",
     "husky": "^7.0.1",
     "image-webpack-loader": "^8.0.1",
     "json-loader": "^0.5.4",
     "moment": "^2.22.2",
-    "postcss": "^8.2.0",
-    "postcss-cli": "^9.0.1",
+    "postcss": "^8.4.5",
+    "postcss-cli": "^9.1.0",
     "postcss-color-function": "^4.1.0",
     "postcss-custom-media": "^8.0.0",
-    "postcss-each": "^0.10.0",
+    "postcss-each": "^1.1.0",
     "postcss-import": "^14.0.0",
-    "postcss-loader": "^4.1.0",
+    "postcss-loader": "^6.2.1",
     "postcss-nested": "^5.0.3",
-    "postcss-preset-env": "^6.7.0",
+    "postcss-preset-env": "^7.2.0",
     "postcss-pseudoelements": "^5.0.0",
-    "postcss-reporter": "^7.0.2",
+    "postcss-reporter": "^7.0.5",
     "prettier": "2.5.1",
-    "pretty-quick": "3.1.2",
+    "pretty-quick": "3.1.3",
     "proptypes": "^1.1.0",
     "rimraf": "^3.0.0",
     "storybook-addon-designs": "^6.0.0",
-    "style-loader": "^2.0.0",
-    "webpack": "^4.1.1"
+    "style-loader": "^3.3.1",
+    "webpack": "^5.65.0"
   },
   "resolutions": {
-    "terser-webpack-plugin": "3.0.6"
+    "immer": "9.0.6",
+    "**/trim": "1.0.0",
+    "prismjs": "1.25.0",
+    "glob-parent": "6.0.1",
+    "browserslist": "4.19.1",
+    "trim-newlines": "3.0.1",
+    "color-string": "1.5.5",
+    "semver-regex": "3.1.3",
+    "nth-check": "2.0.1",
+    "ansi-regex": "5.0.1"
   },
```

Comment:
Should we document somewhere for each of these dependencies why we did this and when they can be removed again? Otherwise these will never get cleaned up after the vulnerabilities are fixed (if ever)

Reply:
Indeed, I don't think we'll move forward with these fixed resolutions, I'd rather wait for the main dependency to update / find a work around to not use the dependency that requires these anymore